### DISCUSSION TITLE
How to decode URL id for this?

### DISCUSSION THREAD

**Discussion by hendrikjap** [Budibase/budibase#3987] [Help]
Using sample for parent - child, I am using supabase postgres. However, when I want to pass URL id from the parent, I am unable to do so. After some observation, I realised 

![image](https://user-images.githubusercontent.com/55825618/148913654-2cfa4866-80a7-4941-9566-63b56135ab5e.png)

I just want to get "GSEX9999". How do I accomplish this?

**Comment by diogenesbrussels** [comment]
Hey @ely2727,

I think I had i similar issue, in fact you should use the {{ field.id }} and not {{ field._id }} in your url.
Let me know if I'm correct.

Cheers

**Reply by mjashanks** [reply]
That right, thanks @diogenesbrussels - when you link to this screen, use `/orders/{{ field.your_primary_key_column}}` instead of `_id`

**Reply by hendrikjap** [reply]
Thanks @diogenesbrussels @mjashanks !

**Reply by hendrikjap** [reply]
![image](https://user-images.githubusercontent.com/55825618/148940684-ab1cc3a9-3e50-445e-bce5-1a30d11d26a4.png)

![image](https://user-images.githubusercontent.com/55825618/148940710-35365b1d-7193-45f2-a5c4-d276c9b4d541.png)

![image](https://user-images.githubusercontent.com/55825618/148940793-4929bd24-4637-439e-a175-d315c9a8fb44.png)

![image](https://user-images.githubusercontent.com/55825618/148940849-707cb8a5-3f18-427c-9f4f-f3b614dbb7ea.png)

@mjashanks , somehow it's showing error. Why could it be?

**Reply by mjashanks** [reply]
@ely2727 where are you seeing this? On saving a new record? 

Could you open up your browser's developer tools, go to the network tab, and try again - let me know if you see any failed requests in there and tell me if it returns an error.

**Reply by hendrikjap** [reply]
Note: this is using Supabase. 

![image](https://user-images.githubusercontent.com/55825618/149030655-8a6267a3-037f-4534-bed8-354bb5c9ed2b.png)

![image](https://user-images.githubusercontent.com/55825618/149030686-5e50b974-78de-4434-8634-d5edb19298ab.png)

![image](https://user-images.githubusercontent.com/55825618/149030712-933f5615-676d-4f10-a661-c2c1a703b593.png)

[lineitemRows.txt](https://github.com/Budibase/budibase/files/7850252/lineitemRows.txt)

**Reply by mjashanks** [reply]
Hi @ely2727 - apologies, I'm just getting back to this. Can you tell me what you see in the `Network > response` tab ....

![image](https://user-images.githubusercontent.com/3524181/149349589-8d8842f4-b708-4e13-808f-43701ffe083d.png)

Also, are you able to create rows in this table vai the `Data` section? 
 
![image](https://user-images.githubusercontent.com/3524181/149349905-d096253f-08fa-4d55-9e2c-473c76af7681.png)

I know Supabase but have not used it... to they package a certain version of Postgres? Do you know what it is?

**Reply by hendrikjap** [reply]
Hi @mjashanks , I rebuilt my tables all from budibase and it is WORKING! Sorry I didn't manage to report what you requested as I had deleted the data.
